cli.js

```diff
@@ -32,7 +32,7 @@ if (!asyncapiFile) {
   program.help(); // This exits the process
 }
 if (!version) {
-  version = '2.1.0';
+  version = '2.2.0';
```

Comment:
Is this related to fixing security vulnerabilities? 🤔

Reply:
No but while testing cli I noticed that without specifying the version it converts the spec to 2.1.0 and not to the latest version 2.2.0. We overlooked this in the previous release :)

Reply:
Oh ok. I'd say you should be opening another PR for that change but I'm not going to block you for that. Just please take this into account for future PRs. If we had to revert this PR for some reason, we could forget about keeping this change that's not related to the purpose of the PR.

Reply:
Right, I will open another PR with that. Could you accept again? :) Thanks!